src/pages/core/addons.module.css

```
@@ -0,0 +1,34 @@
+/* /core/addons — page-specific. Shared base composed from PaperLayout. */
+
+.section     { composes: section     from '../../components/PaperLayout/paper.module.css'; }
+.sectionAlt  { composes: sectionAlt  from '../../components/PaperLayout/paper.module.css'; }
```

Stylelint (17.12.0)

[error] Expected "sectionAlt" to be "sectionalt" (value-keyword-case)

(value-keyword-case)